profile_change_goal_fragment.xml

```diff
@@ -0,0 +1,214 @@
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="20dp"
+        android:layout_marginBottom="16dp"
+        android:lineSpacingExtra="5.6sp"
+        android:text="@string/information"
+        android:textColor="@color/color_primary_text"
+        android:textSize="14sp"
+        app:layout_constraintBottom_toTopOf="@+id/volunteerHR_text"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <TextView
+        android:id="@+id/donation_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="20dp"
+        android:layout_marginTop="14dp"
+        android:layout_marginBottom="16dp"
+        android:lineSpacingExtra="5.6sp"
+        android:textColor="@color/color_primary_text"
+        android:textSize="14sp"
+        app:layout_constraintBottom_toTopOf="@+id/information_text"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/Here_to_offer_text"
+        tools:text="Donations" />
+
+    <TextView
+        android:id="@+id/volunteerHR_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="20dp"
+        android:layout_marginBottom="34dp"
+        android:lineSpacingExtra="5.6sp"
+        android:text="@string/volunteer_hours"
+        android:textColor="@color/color_primary_text"
+        android:textSize="14sp"
+        app:layout_constraintBottom_toTopOf="@+id/here_to_request_test"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <ImageView
+        android:id="@+id/profile_donation_back_arrow"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="20dp"
+        android:layout_marginTop="60dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@drawable/ic_back_arrow" />
+
+    <TextView
+        android:id="@+id/other_details_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="20dp"
+        android:textSize="12sp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/other_help_text"
+        tools:text="I need assistance getting groceries, \n medicine, etc." />
+
+    <TextView
+        android:id="@+id/medical_help_details"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="20dp"
+        android:textSize="12sp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/medical_help_text"
+        tools:text="I have symptoms of COVID-19." />
+
+    <ImageView
+        android:id="@+id/img_donation_box"

```

Comment:
I think we should use <Checkbox> component, see [this](https://material.io/develop/android/components/checkboxes) and [this](https://developer.android.com/guide/topics/ui/controls/checkbox). Also, we should use TextInputLayout for alignment purposes if it's possible with <CheckBox>.  You can check examples on Signin and Signup fragments. This applies to all checkboxes.